game/scripts/npc/abilities/obsidian_destroyer_arcane_orb_oaa.txt

```diff
@@ -32,7 +32,7 @@
       "01"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "mana_pool_damage_pct"                            "6 7 8 9 10 11"
+        "mana_pool_damage_pct"                            "7 8 9 10 11 12" //OAA
```

Comment:
I don't think this will change anything. Maybe increase int steal by 1 at all levels instead or make the duration scale a bit.

Reply:
This is because the spell does magic damage. The percentage remained the same despite it being changed to magic damage